### DISCUSSION TITLE
[Feature] Search for a location instead of moving the map

### DISCUSSION THREAD

**Discussion by roscio1975** [immich-app/immich#7457] [Feature Request]
### The feature

I would like to search box for a location instead of moving on the map
![add location](https://github.com/immich-app/immich/assets/380012/cceb25e8-8cc6-42ce-b12a-0f677e8804b7)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#7139
